### PR TITLE
Try to fix unicode normalisation test in Windows.

### PR DESCRIPTION
**Pull request by scoder**

### CODE REVIEW

File: Cython/TestUtils.py [hunk 1]
```diff
@@ -189,28 +190,25 @@ def unpack_source_tree(tree_file, dir=None):
         dir = tempfile.mkdtemp()
     header = []
     cur_file = None
-    f = open(tree_file)
-    try:
+    with open(tree_file, mode='rb') as f:
         lines = f.readlines()
-    finally:
-        f.close()
     del f
```
Comment (h-vetinari) [inline Cython/TestUtils.py]: This line probably needs to go as well...

Comment (scoder) [reply Cython/TestUtils.py]: Right.

**Comment by scoder** [comment]
Closing and reopening since github apparently lost track of the branch head.

**Comment by scoder** [comment]
Merging this since it's "good enough" and already fixes a lot of things.

**Pull request merged**